src/rgw/rgw_rados.cc

```diff
@@ -3764,6 +3764,22 @@ int RGWRados::init_rados()
     }
   }
 
+  {
+    map<string,string> metadata;
+    metadata["pid"] = stringify(getpid());
+    char hostname[1024] = {0};
+    gethostname(hostname, sizeof(hostname));
+    metadata["host"] = hostname;
+    metadata["num_handles"] = handles.size();
+    metadata["zonegroup_id"] = zonegroup_id;
+    metadata["zone_name"] = zone_name;
+    string name = g_conf->name.get_id();
+    if (name.find("rgw.") == 0) {
+      name = name.substr(4);
```

Comment:
is name the attribute which should indicate that (e.g.) this is a "radosgw" or "nfs-ganesha" instance?  a new field? 
for nfs-ganesha, there is other info that we'd eventually be able to publish (e.g., exports)